### ISSUE TITLE
A question on week_0 

### ISSUE THREAD

**Issue by chloamme** [graviraja/MLOps-Basics#23]
Hello Raviraj, your great work is so helping me.

I installed all packages using requirements.txt. And I trained a model without any issue. 
But I have an issue while I do inference sentences.

In my case, every sentence gets the same results. (almost same score)
Could you check it out? Thanks!

![image](https://user-images.githubusercontent.com/78716704/134186328-344b154f-0d3d-4434-a49a-190962a31ae6.png)
![image](https://user-images.githubusercontent.com/78716704/134186360-906df3c6-47f4-44bc-be19-157f168867c9.png)

**Comment by ravirajag** [comment]
Hi @chloamme can you paste the training script also.

**Comment by chloamme** [comment]
This is the training script.

```
$ git clone https://github.com/graviraja/MLOps-Basics.git
$ cd MLOps-Basics/week_0_project_setup/
$ pip install -r requirements.txt 
$ python train.py 
$ ll models/epoch\=1-step\=535.ckpt 

After I got a checkpoint, I edited the inference.py file using the name of the ckpt file I've obtained.
And I added a few more examples to inference on the model.
$ python inference.py 
```

And my environment is,
- CUDA Version: 11.0
- Python 3.6.12

Tensorboard captures are
![image](https://user-images.githubusercontent.com/78716704/134290791-2bbcc378-ac19-49b2-9d5d-e23ddeb8eb56.png)

**Comment by ravirajag** [comment]
The model is training. As you can see the loss is decreasing. Since the goal is to explore MLOps, not model training, I have done only a basic one. For the model to perform better, either try with a different model (I have used the smallest one to run experiments faster) or tune the hyper-parameters.

**Comment by chloamme** [comment]
I thought maybe the inference was going wrong, with very different inputs giving almost the same scores. 
These are the sentences using the training step; one is for acceptable, and the other is vice versa. 
![image](https://user-images.githubusercontent.com/78716704/134296453-768b4290-ee84-47e6-8256-787721895aad.png)

But, They got similar scores also. I guessed these samples would get distinct scores. So, I was confused.
```
There were killed three men by the assassin. 
         [{'label': 'unacceptable', 'score': 0.3104795813560486}, {'label': 'acceptable', 'score': 0.6895204186439514}]
The critics laughed the play off the stage. 
         [{'label': 'unacceptable', 'score': 0.31048455834388733}, {'label': 'acceptable', 'score': 0.6895154714584351}]
```

I'll tune the hyper-parameters and try again! Thanks!

**Comment by chloamme** [comment]
I changed only the lr value (1e-7), and it works well!! Thank you very much! 😄
